scss/_accordion.scss

```diff
@@ -0,0 +1,146 @@
+//
+// Base styles
+//
+
+.accordion-button {
+  @include font-size($font-size-base);
+  display: flex;
+  align-items: baseline;
+  width: 100%;
+  padding: $accordion-button-padding-y $accordion-button-padding-x;
+  color: $accordion-button-color;
+  text-align: left;
+  background-color: $accordion-button-bg;
+  border: $accordion-border-width solid $accordion-border-color;
+  border-bottom: none;
+  @include border-radius(0);
+
+  &:not(.collapsed) {
+    color: $accordion-button-active-color !important; // stylelint-disable-line declaration-no-important
+    background-color: $accordion-button-active-bg !important; // stylelint-disable-line declaration-no-important
+    &::after {
+      background-image: escape-svg($accordion-button-active-icon);
+      transform: rotate(180deg);
```

Comment:
Would make this variable, could be people want to rotate 90 degrees, or flip the chevron